hooks/boot/13-tuning.rb

```diff
@@ -4,10 +4,11 @@
   TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large']
   TUNING_FACT = 'tuning'.freeze
 
-  app_option(
-    '--tuning',
-    'INSTALLATION_SIZE',
-    "Tune for an installation size. Choices: #{TUNING_SIZES.join(', ')}",
-    :default => get_custom_fact(TUNING_FACT)
-  )
+  app_option('--tuning', 'INSTALLATION_SIZE', "Tune for an installation size. Choices: #{TUNING_SIZES.join(', ')}") do
```

Comment:
Don't you need to pass the value?
```suggestion
  app_option('--tuning', 'INSTALLATION_SIZE', "Tune for an installation size. Choices: #{TUNING_SIZES.join(', ')}") do |value|
```